docs/en/usage-with-symfony.md

```diff
@@ -0,0 +1,58 @@
+# Usage with Symfony
+
+Require migrations and DB driver. Let's use SQLite for this example:
+
+```
+composer require yiisoft/db-migration
+composer require yiisoft/db-sqlite
+```
+
+Configure migrations and database connection in your `config/services.yml`:
+
+```yaml
+Yiisoft\Db\Migration\:
+    resource: '../vendor/yiisoft/db-migration/src/'
+
+Yiisoft\Aliases\:
+    resource: '../vendor/yiisoft/aliases/src/'
```

Comment:
What does it mean? 
Seems like composer stuff

Reply:
That allows using services from the path with autowiring in Symfony DI container.